Automattic/WP-Job-Manager
# Add job search submission button

Fixes #1409 

Adds a "Search Jobs" button to the job filters UI for accessibility purposes, and a filter to hide the button if desired.

## Approach

I decided to add the button directly to the template. This is consistent with the rest of the fields in the template. An alternative would be to add the button using the `job_manager_job_filters_search_jobs_end` action instead.

Some implications of adding directly to the template:

- Existing themes/sites with custom templates will not have this button unless they add it to their template.
- Themes or plugins that adds a similar button using the aforementioned action will need to disable the new button using the new filter (currently Jobify adds a button in this way).

## Testing instructions

- On the "jobs" page, ensure that the filters work when pressing Enter, and when clicking the button.
- Test with several themes, to ensure that the button looks and works ok on them all.
- Ensure that this code snippet hides the button:

```php
add_filter( 'job_manager_job_filters_show_submit_button', '__return_false' );
```
